src/transformers/training_args.py

```diff
@@ -775,6 +775,9 @@ class TrainingArguments:
 
         eval_on_start(`bool`, *optional*, defaults to `False`):
             Whether to perform a evaluation step (sanity check) before the training to ensure the validation steps works correctly.
+
+        use_gather_object(`bool`, *optional*, defaults to `False`):
```

Comment:
Since it is used only for evaluation, probably we should add `eval_` prefix.

Reply:
Thanks!